examples/test-animate-color/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Animate Color</title>
+    <meta name="description" content="Animate Color - A-Frame">
+    <script src="../../dist/aframe.js"></script>
+  </head>
+  <body>
+    <a-scene stats="true">
+      <a-sky color="#afafaf"></a-sky>
+      <a-box color="white" position="2 1 -10">
```

Comment:
Alright I'll look into that and get the working example to run around that method.  Getting caught up at work today too so probably look for this adjustment tomorrow.